docs/en/ingest-management/fips-ingest.asciidoc

```diff
@@ -0,0 +1,118 @@
+[role="xpack"]
+[[fips-ingest]]
+= FIPS mode for Ingest tools
+
+preview::[]
+
+{agent}, {fleet}, {filebeat}, {metricbeat}, and APM Server binaries are built and can be configured to use FIPS 140-2 compliant cryptography.
```

Comment:
@simitt @ycombinator nit: should we say `can be configured` or `are configured`? Aren't the defaults using FIPS compliant crypto for these builds?

Reply:
I think "are configured" is more accurate.